### PR TITLE
57 alpine support

### LINKED ISSUE #57: [IMPROVEMENT] Add support for Alpine guests
We should add Alpine family support

### PR DESCRIPTION
**Pull request by 89luca89**
Fix #57

### CODE REVIEW

File: examples/inventory-test-alpine.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+all:
```
Comment (alegrey91) [inline examples/inventory-test-alpine.yml:1]: You forgot to add the alpine test on Travis

Comment (alegrey91) [reply examples/inventory-test-alpine.yml:1]: Remember also to push the image on the cloud to be pulled during the pipeline

File: examples/inventory-test-alpine.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        pool_name: default
+        ssh_password: root
+        ssh_port: 22
+        ssh_user: root
+        ssh_public_key_file: '~/.ssh/id_rsa.pub'
+      children:
+        master_nodes:
+          hosts:
+            host-vm-1:
+              os_family: Debian
```
[the remark this answers is not in the document]
Comment (89luca89) [reply examples/inventory-test-alpine.yml]: Fixed

File: examples/inventory-test-alpine.yml (hunk 1 above)
Comment (89luca89) [reply examples/inventory-test-alpine.yml:1]: Added

**Pull request merged**